repository_cli/cli/click_options.py

```diff
@@ -9,19 +9,53 @@
 
 import click
 
+
 # -i '{ "identifier": "10.48436/fcze8-4vx33", "scheme": "doi"}'
-option_identifier = click.option(
-    "--identifier",
-    "-i",
-    required=True,
-    help="metadata identifier as JSON",
-)
+def option_identifier(required=False):
+    """Get parameter options for metadata identifier."""
+    return click.option(
+        "--identifier",
+        "-i",
+        required=required,
+        help="metadata identifier as JSON",
+    )
+
 
 # -p "fcze8-4vx33"
-option_pid = click.option(
-    "--pid",
-    "-p",
-    metavar="PID_VALUE",
-    required=True,
-    help="persistent identifier of the object to operate on",
-)
+def option_pid(required=False):
+    """Get parameter options for record PID."""
+    return click.option(
+        "--pid",
+        "-p",
+        metavar="PID_VALUE",
+        required=required,
+        help="persistent identifier of the object to operate on",
+    )
+
+
+# -if "input.json"
+def option_input_file(required=False):
+    """Get parameter options for input file."""
+    return click.option(
+        "--input-file",
+        "-if",
```

Comment:
please stay posix conform. so one hyphen is only one character, two hyphens more then one characters